setup.cfg

```diff
@@ -1,24 +1,26 @@
-[bdist_wheel]
-universal=1
+[bumpversion]
+current_version = 3.0.0
```

Comment:
is this redundant with the version 3.0.0 that appears in another file?

Reply:
I am honestly not sure. I know when I run the `bumpversion {option}` command here, this instance of the version, the `setup.py`, and, the `__init__.py` version strings all get updated so to me it isn't an issue.